stdlib/distutils/command/bdist_msi.pyi

```diff
@@ -12,32 +12,33 @@ if sys.platform == "win32":
         def cancel(self, title, next, name: str = ..., active: int = ...): ...
         def next(self, title, next, name: str = ..., active: int = ...): ...
         def xbutton(self, name, title, next, xpos): ...
-    class bdist_msi(Command):
-        description: str
-        user_options: Any
-        boolean_options: Any
-        all_versions: Any
-        other_version: str
-        if sys.version_info >= (3, 9):
-            def __init__(self, *args, **kw) -> None: ...
-        bdist_dir: Any
-        plat_name: Any
-        keep_temp: int
-        no_target_compile: int
-        no_target_optimize: int
-        target_version: Any
-        dist_dir: Any
-        skip_build: Any
-        install_script: Any
-        pre_install_script: Any
-        versions: Any
-        def initialize_options(self) -> None: ...
-        install_script_key: Any
-        def finalize_options(self) -> None: ...
-        db: Any
-        def run(self) -> None: ...
-        def add_files(self) -> None: ...
-        def add_find_python(self) -> None: ...
-        def add_scripts(self) -> None: ...
-        def add_ui(self) -> None: ...
-        def get_installer_filename(self, fullname): ...
+    if sys.version_info < (3, 11):
```

Comment:
The whole file seems to be gone, not just `class bdist_msi`. Edit `stdlib/VERSIONS`.

Reply:
The wholesale removal of `smtpd` may be reverted; we're waiting on a decision from the Steering Council on that one. https://github.com/python/steering-council/issues/86. I'll just take the changes to the file out for now, since there's not much point adding the `if sys.version_info` check if the whole module is removed.